test/request.test.ts

```diff
@@ -586,8 +588,10 @@ x//0u+zd/R/QRUzLOw4N72/Hu+UG6MNt5iDZFCtapRaKt6OvSBwy8w==
       })
 
       .catch((error) => {
-        expect(error.message).toMatch(/\bsignal\b/i);
-        expect(error.message).toMatch(/\bAbortSignal\b/i);
+        // We can't match on the entire string because the message differs between
+        // Node versions. In v14 and v16, the message just mentions "signal", whereas
+        // in v18 it contains the stringified signal.
+        expect(error.message).toContain("to be an instance of AbortSignal");
```

Comment:
```suggestion
        expect(error.message).toContain("to be an instanceof AbortSignal");
```

Reply:
This is the output I'm seeing in [our tests (Node 16)](https://github.com/octokit/request.js/runs/7838415908?check_suite_focus=true):
```
FAIL test/request.test.ts ([9](https://github.com/octokit/request.js/runs/7838415908?check_suite_focus=true#step:5:10).281 s)
  ● request() › options.request.signal is passed as option to fetch

    expect(received).toContain(expected) // indexOf

    Expected substring: "to be an instance of AbortSignal"
    Received string:    "Expected signal to be an instanceof AbortSignal"

      592 |         // Node versions. In v[14](https://github.com/octokit/request.js/runs/7838415908?check_suite_focus=true#step:5:15) and v[16](https://github.com/octokit/request.js/runs/7838415908?check_suite_focus=true#step:5:17), the message just mentions "signal", whereas
      593 |         // in v18 it contains the stringified signal.
    > 594 |         expect(error.message).toContain("to be an instance of AbortSignal");
          |                               ^
      595 |       });
      596 |   });
      597 |

      at test/request.test.ts:594:31
```

Reply:
Ah, so sorry, I didn't see the failure for v16 😓 Thanks for pointing that out! I've now pushed a version which works across versions using a regex.

Reply:
No problem. It's a bit tricky to have different outputs depending on the Node version.